boot.php

```diff
@@ -15,9 +20,9 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+$filename = getcwd().'/'.$argv[1];
```

Comment:
We've discussed that already, see https://github.com/gabrielrcouto/php-terminal-gameboy-emulator/pull/16#discussion_r53795614

I'm :+1: for this pull request, but I think just `$argv[1]` (without the `getcwd().'/'.`) is enough.